test/Enable.Extensions.Queuing.AzureServiceBus.Tests/ConcurrentQueueClientTests.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Security;
+using Enable.Extensions.Queuing.AzureServiceBus.Internal;
+using Xunit;
+
+namespace Enable.Extensions.Queuing.AzureServiceBus.Tests
+{
+    public class ConcurrentQueueClientTests
+    {
+        private readonly AzureServiceBusQueueClientFactory _factory;
+        private readonly string _queueName;
+
+        public ConcurrentQueueClientTests()
+        {
+            var connectionString = GetEnvironmentVariable("AZURE_SERVICE_BUS_CONNECTION_STRING");
+
+            var factoryOptions = new AzureServiceBusQueueClientFactoryOptions
+            {
+                ConnectionString = connectionString
+            };
+
+            _factory = new AzureServiceBusQueueClientFactory(factoryOptions);
+            _queueName = GetEnvironmentVariable("AZURE_SERVICE_BUS_QUEUE_NAME");
+        }
+
+        [Fact]
+        public void ConcurrentQueueClientsUseSameQueue()
+        {
+            // Arrange
+            var client = _factory.GetQueueReference(_queueName) as AzureServiceBusQueueClient;
+            var anotherClient = _factory.GetQueueReference(_queueName) as AzureServiceBusQueueClient;
+
+            // Assert
+            Assert.True(object.ReferenceEquals(client.Queue, anotherClient.Queue));
```

Comment:
This can be simplified to: `Assert.Assert.ReferenceEquals(client.Queue, anotherClient.Queue);`.

Reply:
Done.